e2e/cypress/support/api/system.js

```diff
@@ -0,0 +1,107 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import merge from 'merge-deep';
+
+import partialDefaultConfig from '../../fixtures/partial_default_config.json';
+
+// *****************************************************************************
+// System
+// https://api.mattermost.com/#tag/system
+// *****************************************************************************
+
+Cypress.Commands.add('apiGetClientLicense', () => {
+    return cy.request('/api/v4/license/client?format=old').then((response) => {
+        expect(response.status).to.equal(200);
+        return cy.wrap({license: response.body});
+    });
+});
+
+Cypress.Commands.add('requireLicenseForFeature', (key = '') => {
+    return cy.apiGetClientLicense().then(({license}) => {
+        expect(license.IsLicensed, 'Server has no Enterprise license.').to.equal('true');
+
+        let hasLicenseKey = false;
+        for (const [k, v] of Object.entries(license)) {
+            if (k === key && v === 'true') {
+                hasLicenseKey = true;
+                break;
+            }
+        }
+
+        expect(hasLicenseKey, `No license for feature: ${key}`).to.equal(true);
+
+        return cy.wrap({license});
+    });
+});
+
+Cypress.Commands.add('requireLicense', () => {
```

Comment:
Need general agreement here. Note that `requireLicense` and `requireLicenseForFeature` didn't start with `api` prefix. Our guideline is to prefix with `api`. However, I feel like it's easier to understand if it starts with "require". Open to suggestion on naming and/or file location if better to move into separate file.

Reply:
I think its fine. We just need to remove the guideline only for this file.

Reply:
I actually prefer to prefix it so that it's consistent across the board. It will be easier to remember and order in code alphabetically if it follows convention. But I understand that would need other files to be updated. Maybe create a ticket to update it later.

Reply:
Thanks for the inputs. Since the opinions are split, I have the luxury to choose (😄) - decided to add api prefix. Feel free to give feedback or change in the future.